scout/advanced-image-analysis.md

```diff
@@ -7,16 +7,25 @@ title: Advanced image analysis
 > **Note**
 >
 > Docker Scout is an [early access](../release-lifecycle.md#early-access-ea)
-> product, and requires a Docker Pro, Team, or Business subscription.
+> product, and requires a Docker Pro, Team, or Business subscription..
+>
+> If you're interested in this integration for your organization and want to
+> learn more, get in touch by filling out the contact form on the
+> [Docker Scout product page](https://docker.com/products/docker-scout){:
+> target="\_blank" rel="noopener" }.
 
 Advanced image analysis is a Docker Scout feature for Docker Hub.
 
-When you activate Advanced image analysis for a repository, new tags are
-automatically analyzed when you push to that repository. Advanced image analysis
-is more than point-in-time scanning - your analysis gets reevaluated
+When you activate advanced image analysis for a repository, Scout analyzes new tags
```

Comment:
I believe having 'advanced' capitalized was correct. It's also now inconsistent here with the rest of the page.

Reply:
```suggestion
When you activate Advanced image analysis for a repository, Scout analyzes new tags
```